externs/lcevc.js

```diff
@@ -0,0 +1,113 @@
+/**
+* @fileoverview Externs for LcevcDil
+* compiler.
+*
+* @externs
+*/
+
+var libDPIModule = {};
+var LcevcDil = {};
+
+/**
+* LCEVC DIL constructor
+* @constructor
+*/
+LcevcDil.LcevcDIL = class {
+  /**
+     *  @param {HTMLVideoElement} media
+     *  @param {HTMLCanvasElement} canvas
+     *  @param {LcevcDil.LcevcDilConfig} dilConfig
+     */
+  constructor(media, canvas, dilConfig) {
+    /** @type {number} */
+    this.aspectRatio;
+
+    /** @type {boolean} */
+    this.isFullscreen;
+  }
+
+  /**
+   * Append the fMP4 fragments
+   * after they are appended to Media Source Extensions
+   * SourceBuffer. Here the lcevc data will be extracted and manged in line with
+   * the ranges given by the SourceBuffer.buffered() call result
+   *
+   * @param {!BufferSource} data fMP4 fragment.
+   * @param {string} type Type of the fragment.
+   * @param {number} level Level (Variant) that the fragment belongs to.
+   */
+  appendBuffer(data, type, level) {}
+
+  /**
+   * Set current variant as level to the LCEVC decoder
+   * @param {!number} level The variant of the fragment.
```

Comment:
"level" has no meaning to us.  What do you mean by "level"?  I see in the code that it is an index into the track list.  Is it just an arbitrary numerical identifier that you intend to be stable?

Reply:
"level" is the variant ID that is used by the lcevcDIL to compare the buffers pushed to its respective variant. So, if the variant changes, we need to know, so that we can handle resolution changes etc.